test/e2e/kubernetes/node/node.go

```diff
@@ -74,14 +74,22 @@ type List struct {
 // AreAllReady returns a bool depending on cluster state
 func AreAllReady(nodeCount int) bool {
```

Comment:
The purpose of these changes is to make this func more scrupulous: instead of "looking for specific failure conditions and returning true otherwise" we are now "looking for specific success conditions and returning false otherwise".